charts/rawls/values.yaml

```diff
@@ -81,3 +81,14 @@ deploymentDefaults:
         periodSeconds: 10
         failureThreshold: 30 # 5 minutes before restarted
         successThreshold: 1
+    startup:
```

Comment:
@choover-broad This file has probes define. However, there's no mention of them in the [deployment file](https://github.com/broadinstitute/terra-helm/blob/master/charts/rawls/templates/deployments.yaml).

Reply:
b54e7e2